pkg/sippyserver/server.go

```diff
@@ -656,6 +659,18 @@ func (s *Server) jsonComponentReportTestDetailsFromBigQuery(w http.ResponseWrite
 	api.RespondWithJSON(http.StatusOK, w, outputs)
 }
 
+func (s *Server) parseReleaseTime(timeStr string) (time.Time, error) {
+	releaseTime, err := time.Parse(time.RFC3339, timeStr)
+	if err != nil {
+		return releaseTime, err
+	}
+	now := time.Now().UTC()
+	if s.crTimeRoundingFactor != 0 && releaseTime.Year() == now.Year() && releaseTime.YearDay() == now.YearDay() {
```

Comment:
`now().Format("2006-01-02") == releaseTime.Format("2006-01-02")` is a little more readable to me, I always forget YearDay is not the day of the month